apps/obs-web/package.json

```diff
@@ -39,10 +39,11 @@
     "pouchdb-authentication": "^1.1.3",
     "pouchdb-browser": "^7.0.0",
     "primeicons": "^2.0.0",
-    "primevue": "^2.2.0",
+    "primevue": "^2.4.0",
     "quasar": "1.8.5",
     "quasar-calendar": "^1.0.0-beta.1",
     "register-service-worker": "^1.5.2",
+    "v-calendar": "^2.2.3",
```

Comment:
v-calendar looks pretty slick.  curious whether a specific feature drove you to it.  Is this implemented for a specific purpose, or testing to potentially replace the prime calendar?

Reply:
Prime vue calendar acts weird when inputting date info. For example, when I'm trying to type in the 12th this is what ends up happening. So using another date component. Prime calendar does works fine though when clicking on dates. Thinking of potentially replacing the other prime calendars, but tried replacing the one that appears when editing the data table the formatting was off so got to figure that out first. So maybe we can slowly migrate over but the other prime calendars seem to be working fine now.

Also not a fan of quasar calendar because the calendar doesn't pop up when you click on the input box. Also you are not able to input the date and see the calendar adjust at the same time, you can only input date info or view the calendar not both at the same time.

![image](https://user-images.githubusercontent.com/44685558/109196224-498d0600-7750-11eb-90de-aaf381eead24.png)

Reply:
cool, let me know what you decide.  Would be great if we could get a single calendar component to work across all of boatnet.

Reply:
yes agree! yes I will let you know, but feel free to use this v-calendar in the future for anything you're building def like it better than any of the other calendars we have so far.